dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -78,6 +82,12 @@ public Options getOptions() {
                     .argName("FILE")
                     .required()
                     .build());
+                options.addOption(Option.builder().longOpt(MAPPING_FILE)
+                    .desc("mapping description as Groovy file")
```

Comment:
And give more detail in the description

Reply:
Would "Dynamic models description as a Groovy file: defines the dynamic models to be associated to chosen equipments of the network" be enough?

Reply:
Description changed by Agnes proposal